### PR TITLE
make post on front page message more intuitive

### PR DESCRIPTION
**Pull request by Xodarap**
See discussion here: https://cea-core.slack.com/archives/C75BWSNBH/p1651179231355559

### CODE REVIEW

**Comment by Raemon** [comment]
Looks like this still needs forum-gating

File: packages/lesswrong/components/posts/PostsPage/PostActions.tsx [hunk 1]
```diff
@@ -172,6 +173,11 @@ const PostActions = ({post, closeMenu, classes}: {
 
   const isRead = (post._id in postsRead) ? postsRead[post._id] : post.isRead;
 
+  const defaultLabel = forumSelect({
```
Comment (Raemon) [inline packages/lesswrong/components/posts/PostsPage/PostActions.tsx:176]: I think "defaultLabel" is the wrong word for the variable which is "either the default label, or a different forum-specific label". I'd maybe say "frontpageMessage" or something.

(Also the fact that the same code is duplicated triggers my "should this just be abstracted?" sensors, although I also do think there's risk from having different UI elements share code, which can result in future devs being surprised when changing something in one place changes it in another place that might evolve UI-wise in different directions)

**Comment by Raemon** [comment]
I left a comment, which I think would be nice-to-fix but not essential. Approving.

**Review by Raemon — APPROVED** [review]

File: packages/lesswrong/components/posts/SubmitToFrontpageCheckbox.tsx [hunk 2]
```diff
@@ -100,7 +100,11 @@ class SubmitToFrontpageCheckbox extends Component<SubmitToFrontpageCheckboxProps
   }
 
   render() {
-    const { classes, label='Moderators may promote to Frontpage', tooltip } = this.props
+    const defaultLabel = forumSelect({
+      EAForum:'Post on Frontpage by default',
```
Comment (s-cheng) [inline packages/lesswrong/components/posts/SubmitToFrontpageCheckbox.tsx:104]: Not to overcomplicate this, but I find this wording quite confusing. From the perspective of a new user, I wouldn't understand when the "default" happens, and what happens when the "default" doesn't happen. (TBH I find "Moderators may promote to Frontpage" confusing as well, but less so.)

I think I like the "This post may appear on the Frontpage" suggestion the best.

**Pull request merged**